src/lib/markup/photodiode.js

```diff
@@ -39,11 +39,11 @@ const pdSpotEncode = (taskCode) => {
     }
 
 		if (config.USE_PHOTODIODE) {
-				const blinkTime = 40
-				let numBlinks = taskCode
-		    if (taskCode < eventCodes.open_task) numBlinks = 1;
-				repeat_pulse_for(blinkTime, numBlinks)
-				if ( ipcRenderer ) ipcRenderer.send('trigger', taskCode)
+      const blinkTime = 40
+      let numBlinks = taskCode
+      if (taskCode < eventCodes.open_task) numBlinks = 1;
+      repeat_pulse_for(blinkTime, numBlinks)
+      if ( ipcRenderer ) ipcRenderer.send('trigger', { taskCode })
```

Comment:
is the on `trigger` set up for this change? why make this an object?

Reply:
Yes I changed that. I used to pass in USE_EVENT_MARKER here , but I didn't change it back.